elements/lisk-p2p/src/peer/outbound.ts

```diff
@@ -165,6 +167,13 @@ export class OutboundPeer extends Peer {
 			});
 		});
 
+		outboundSocket.on(
+			EVENT_PING,
+			(_: undefined, res: (_: undefined, data: string) => void) => {
+				res(undefined, 'pong');
```

Comment:
EVENT_PING is const, so we should make this `const` too

Reply:
I will change it to empty string as it's just a placeholder string (there is no `pong` event)

Reply:
I think it would be better to return pong for convention rather than returning empty string, or maybe follow socket cluster protocol?